src/analyse.js

```diff
@@ -0,0 +1,17 @@
+const average = (arr) => arr.reduce((acc, num) => acc + num, 0) / arr.length;
+
+const min = (arr) => Math.min(...arr);
+
+const max = (arr) => Math.max(...arr);
+
+const analyse = (arr) => {
+  const analysed = {
+    average: average(arr),
+    min: min(arr),
+    max: max(arr),
+    length: arr.length,
+  };
+  return analysed;
+};
```

Comment:
- [x] **`Optional`** You could move `average`, `min`, and `max` into the `analyse` function. It's always better to use pure functions where possible.

```suggestion
const analyse = (arr) => {
  const average = (arr) => arr.reduce((acc, num) => acc + num, 0) / arr.length;
  const min = (arr) => Math.min(...arr);
  const max = (arr) => Math.max(...arr);
 
  const analysed = {
    average: average(arr),
    min: min(arr),
    max: max(arr),
    length: arr.length,
  };
  return analysed;
};
```